gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
At the moment, **gcm** is still main subjective in the package and **fcm** is still optional. So **defaultEndpoint** is GCM's. Even if we change the package name, it is not the time yet now. Because there is not the definite difference between **gcm** and **fcm** except a endpoint.